lib/gears/math.lua

```diff
@@ -79,5 +79,16 @@ function gmath.round(x)
     return math.floor(x + 0.5)
 end
 
+--- Round a number to a 10s place
+-- @class function
+-- @name roundf
+-- @tparam number v
+-- @tparam[opt=1] number bracket 10s place to round to
+function gmath.roundf(v, bracket)
+	-- From lua-users.org SimpleRound
+	local function sign(v) return (v >= 0 and 1) or -1 end
+	bracket = bracket or 1
```

Comment:
Is the default of 1 sensible? In that case this just rounds away from zero (while gears.math.round rounds towards positive infinity).
Which raises the question if `gears.math.round` is wrong and should be fixed. C99's `round()` rounds away from zero as well and this behaviour would allow to implement this function as `gears.math.round(v / bracket) * bracket`.